clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -396,6 +396,23 @@ message Query {
 }
 
 message SearchRequest {
+
+    // Specification for kNN vector search query
+    message KnnQuery {
```

Comment:
Should we define this message outside SearchRequest since that's how it's done for all of the other parameters here?